src/github.ts

```diff
@@ -56,6 +56,65 @@ export async function createFailureIssue(
   return resp.data.number;
 }
 
+/**
+ * Comment on a GitHub issue (or pull request).
+ * @param {Octokit} client the GitHub client to use that is authenticated as the GitHub installation.
+ * @param {string} org the GitHub organization where the issue is to add a comment.
+ * @param {string} repo the GitHub repository where the issue is to add a comment.
+ * @param {number} id the issue ID.
+ * @param {string} message the content of the comment.
+ * @return {Promise<boolean>}
+ */
+export async function addComment(
```

Comment:
@dhaiducek could you please remove this unused function?

Reply:
I plan to use it in the future, but I can remove it if that's preferred:
https://github.com/stolostron/magic-mirror/pull/36#issuecomment-1553693440